core/domain/stats_domain.py

```diff
@@ -1497,14 +1701,20 @@ class StateAnswersCalcOutput:
     """
 
     def __init__(
-            self, exploration_id, exploration_version, state_name,
-            interaction_id, calculation_id, calculation_output):
+        self,
+        exploration_id: str,
+        exploration_version: int,
+        state_name: str,
+        interaction_id: str,
+        calculation_id: str,
+        calculation_output: object
```

Comment:
Why is this `object`?

Reply:
It it is typed as `AnswerCalculationOutput` mypy says `Incompatible types in assignment (expression has type "MockCalculationOutputObjectWithUnknownType", variable has type "AnswerCalculationOutput")  [assignment]`

Reply:
Where does this happen, this seems like it happens in tests, if that is so the you can add an ignore into the tests with a proper explanation.

Reply:
Done (added comment to tests)